specification/components/r4/schemas/responses/R4-Forbidden.yaml

```diff
@@ -2,10 +2,10 @@ description: |
   Where status code 403 (Forbidden) is returned then an [NHSDigital-OperationOutcome](https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome) will be included in the body, as detailed below.
   Check diagnostics property for specific information regarding the error.
   
-  | issue.details.coding.code | issue.code | Description                                                                        |
-  | ------------------------- | ---------- | ---------------------------------------------------------------------------------- |
-  | REC_FORBIDDEN             | forbidden  | A call attempts to access or operate upon a resource without proper authorisation. |
-  | NO_RELATIONSHIP           | forbidden  | The caller has no legitimate relationship with the resource.                       |
+  | issue.details.coding.code | issue.code | Coding System                                                      | Description                                                                        |
+  | ------------------------- | ---------- | ------------------------------------------------------------------ | ---------------------------------------------------------------------------------- |
+  | REC_FORBIDDEN             | forbidden  | [BaRS Error Code](https://fhir.nhs.uk/CodeSystem/http-error-codes) | A call attempts to access or operate upon a resource without proper authorisation. |
```

Comment:
OK so the systems as hyperlinks. You can't follow them, but eventually you may be able to. Certainly more presentable than putting the URI inline.

Reply:
I guess we could include the simplifier link instead here for the moment?